lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/config/ConfigUtils.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.base.config;
+
+import com.google.common.collect.ImmutableSet;
+import io.airlift.configuration.ConfigPropertyMetadata;
+
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public final class ConfigUtils
+{
+    private ConfigUtils() {}
+
+    public static Set<String> getSecuritySensitivePropertyNames(Map<String, String> config, Set<ConfigPropertyMetadata> usedProperties)
```

Comment:
the method name is inverted from logic. We remove all properties marked as "security sensitive" from the set and return "safe properties".

Maybe `filterSecuritySensitiveProperties`?

Reply:
> We remove all properties marked as "security sensitive"

We remove all properties that are NOT marked as "security sensitive"

Reply:
Double negation confused me 😞 

Thanks for checking.